x-pack/plugins/security/server/security_checkup/check_cluster_data.ts

```diff
@@ -13,18 +13,22 @@ export const createClusterDataCheck = () => {
   return async function doesClusterHaveUserData(esClient: ElasticsearchClient, log: Logger) {
     if (!clusterHasUserData) {
       try {
-        const indices = await esClient.cat.indices({
-          format: 'json',
-          h: ['index', 'docs.count'],
-        });
-        clusterHasUserData = indices.some((indexCount) => {
+        const { indices = {} } = await esClient.indices.stats();
```

Comment:
__issue:__ I think calling `_stats` call without any parameters is a way to expensive, is there any reason we cannot narrow it down with something like this? `GET /_stats/docs?filter_path=indices.*.total.docs.count` (`esClient.indices.stats` should support the relevant parameters)?